File: metnum.php
<?php

function f($x){
  $f = sqrt($x) + 1 - (1/2*exp($x));
  return $f;
};

echo "Low : ";
$a = trim(fgets(STDIN));
echo "High : ";
$b = trim(fgets(STDIN));
echo "Stop : ";
$n = trim(fgets(STDIN));

$x = 0;
while($x <= $b) {
    echo $x ,"   ", f($x) ,"\n";
    $x = $x + $n;
}
?>

File: metnum.py
import math as m

def f(x):
    return m.sqrt(x) + 1 - (1/2*m.exp(x))
a = int(input('Low : '))
b = int(input('High : '))
n = float(input('Stop : '))
x = 0

print('======================================')
while x <= b:
    print(round(x,3) ,"  ", str(f(x)))
    x += n
print('======================================')
